docs/timelion/getting-started/timelion-conditional.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[timelion-conditional]]
+=== Using conditional logic and tracking trends
+
+In this section you will learn how to modify a series with conditional logic and trend your data with a moving average. This is helpful to easily distinguish outliers and detect trends over time.
```

Comment:
s/"distinguish outliers and detect trends over time"/"detect outliers and patterns over time"